Add docs for the --bucket_path artifact flag (#269)
gridai/grid-docs: docs/features/runs/2_Analyzing Runs/3_artifacts.md

grid artifacts runname-expN
```
:::note
If you have additional questions about Runs, please visit the [FAQ](https://docs.grid.ai/features/runs/faq). This section is periodically updated with common questions from the Grid community.
:::

### See where your artifacts are stored in the cloud (BYOC clusters only)

If you launched a run on a [BYOC cluster](../../../platform/2_Custom%20Cloud%20Credentials/1_byoc.md)
and would like to retrieve the cloud storage location for the artifacts rather than download them,

> use the `grid artifacts --bucket_path <run or experiment>`.

Reviewer: ```suggestion
use the `grid artifacts --bucket_path <run or experiment>` command.
```

> It will display a table showing where each experiment stored its artifacts:

Reviewer: ```suggestion
That command will display a table showing where each experiment stored its artifacts:
```

:::note

> If you have additional questions about Runs, visit the [FAQ](https://docs.grid.ai/features/runs/faq). The section is periodically updated this with common questions from the Grid community.

Reviewer: ```suggestion
If you have additional questions about Runs, please visit the [FAQ](https://docs.grid.ai/features/runs/faq). This section is periodically updated with common questions from the Grid community.
```

:::

### See where your artifacts are stored in the cloud (BYOC clusters only)

If you launched a run on a [BYOC cluster](../../../platform/2_Custom%20Cloud%20Credentials/1_byoc.md)
and would like to get the cloud storage location for the artifacts instead of downloading them,
use the `grid artifacts --bucket_path <run or experiment>`.

> It will display a table showing where each experiment stored its artifacts:

Reviewer: Can we clarify/explicitly name what 'it' refers to here?

Reviewer 2: @RobertLaurella It should be something along the lines of "the output of that command will display a table showing where each experiment stored its artifacts" 
What do you think?

Reviewer: I shortened it to make it simpler - I'm happy with how it reads now if this makes sense & is accurate!
